src/app/feature/consultant/services/export-profile/export-profile.service.ts

```diff
@@ -0,0 +1,299 @@
+import { Document, Paragraph, Packer, TextRun, Table } from 'docx';
+
+export class ExportProfile {
```

Comment:
Grouping this class with services and calling it such is a misnomer and may cause confusion. Services are injected and typically used to communicate between components whereas classes are instantiated and cannot share between components. Because this class is only used in the header section my recommendation would be to move this file and it's associated spec to `src/app/feature/consultant/components/consultant-header/export-profile.ts`.